app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -21,6 +22,8 @@ import org.mozilla.fenix.ui.robots.navigationToolbar
  * Test Suite that contains tests defined as part of the Smoke and Sanity check defined in Test rail.
  * These tests will verify different functionalities of the app as a way to quickly detect regressions in main areas
  */
+
+@Ignore("Temp disable for triggering a native Gecko crash - https://github.com/mozilla-mobile/fenix/issues/11642")

```

Comment:
Could we ignore the test instead of the class? the idea is to add more tests to this class shortly...we can do that change in future PR...as you prefer...

Reply:
Yes, feel free to drop it to the test level when you update the test next.

Reply:
I will keep this in my mind while working on the next test. Thanks for disabling this test for now.